lib/Cake/Network/Email/CakeEmail.php

```diff
@@ -1717,6 +1721,27 @@ protected function _renderTemplates($content) {
 			$rendered[$type] = implode("\n", $rendered[$type]);
 			$rendered[$type] = rtrim($rendered[$type], "\n");
 		}
+
+		/* Embed images inline in html templates */
+		if (!empty($rendered['html'])) {
```

Comment:
I see.

```
$args['content'] = 'Hola mundo<img src="file:c:\\var\\Portapapeles01.jpg"';
$args['content1'] = 'Hola mundo<img src="file:c:\\var\\Portapapeles01.jpg"';
$args['content2'] = 'Hola mundo<img src="file:c:\\var\\Portapapeles01.jpg"';
$args['content3'] = 'Hola mundo<img src="file:c:\\var\\Portapapeles01.jpg"';
$args['content4'] = 'Hola mundo<img src="file:c:\\var\\Portapapeles01.jpg"';
$args['content5'] = 'Hola mundo<img src="file:c:\\var\\Portapapeles01.jpg"';
$args['content6'] = 'Hola mundo<img src="file:c:\\var\\Portapapeles01.jpg"';
$args['content7'] = 'Hola mundo<img src="file:c:\\var\\Portapapeles01.jpg"';
$args['content8'] = 'Hola mundo<img src="file:c:\\var\\Portapapeles01.jpg"';
$args['content9'] = 'Hola mundo<img src="file:c:\\var\\Portapapeles01.jpg"';
$email->viewVars($args);

$loops = 10000000;
$t1 = explode(' ', microtime());
for ($i = 0; $i < $loops; $i++) $data = json_encode($this->viewVars);
$t2 = explode(' ', microtime());
$t = $t2[1] - $t1[1];
$t += $t2[0] - $t1[0];
debug("json:\t" . $t);

$t1 = explode(' ', microtime());
for ($i = 0; $i < $loops; $i++) $data = print_r($this->viewVars, true);
$t2 = explode(' ', microtime());
$t = $t2[1] - $t1[1];
$t += $t2[0] - $t1[0];
debug("print_r:\t" . $t);

$t1 = explode(' ', microtime());
for ($i = 0; $i < $loops; $i++) $data = serialize($this->viewVars);
$t2 = explode(' ', microtime());
$t = $t2[1] - $t1[1];
$t += $t2[0] - $t1[0];
debug("serialize:\t" . $t);



```
The results (10 000 000 loops):
json:          51.134730 seconds
print_r:       62.491546 seconds
serialize:     17.071474 seconds

I'm try to use serialize function... <2usec for each loop...